src/doorman/TelnetProtocol.cpp

```diff
@@ -0,0 +1,171 @@
+#include "TelnetProtocol.hpp"
+
+#include <arpa/telnet.h>
+
+#include <algorithm>
+#include <cstring>
+
+static bool supports_ansi(const char *src) {
+    static const char *terms[] = {"xterm",      "xterm-colour", "xterm-color", "ansi",
+                                  "xterm-ansi", "vt100",        "vt102",       "vt220"};
```

Comment:
Please add "terminator" :-)

Reply:
LOL. this list is frankly abominable: it's the one we inherited and really should use `termcap`.  We don't even use it (See #76)

Done though...